### PR TITLE
Updated tutorial (day1) for 1.x version

### PR DESCRIPTION
**Pull request by mickaelandrieu**
ping @eric-chau  please review :)

### CODE REVIEW

**Comment by mickaelandrieu** [comment]
@clementbonfils any chance to publish this text directly on backbee.com ? what do you think ?

**Comment by cordoval** [comment]
no sir, I am the team reviewer :blush:  :+1:

File: doc/day1.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ Installation
 
 Open a command line interface and get BackBee:
 
-    $ composer create-project "backbee/backbee-standard" /path/to/your/folder "0.11.*@dev"
+    $ composer create-project "backbee/backbee-standard" /path/to/your/folder "1.0" --no-interaction
```
Comment (cordoval) [inline doc/day1.md]: could be just -n but good

File: doc/day1.md [hunk 2]
```diff
@@ -100,12 +106,16 @@ If everything is correctly installed, you should see the BackBee welcome message
 
 ![BackBee welcome message](http://i.imgur.com/rWY8IQk.png "BackBee welcome message")
 
+
+> *If you can't access BackBee welcome message, check both ``cache`` and ``log`` folders have the correct rights.*
```
Comment (cordoval) [inline doc/day1.md]: can't see

Comment (cordoval) [inline doc/day1.md]: check that both ...

**Comment by eric-chau** [comment]
thank you @cordoval =)

**Comment by mickaelandrieu** [comment]
@cordoval updates done, thank you for review :+1:

**Pull request merged**